modules/olm-bundle-format.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding_olm/olm-packaging-formats.adoc
+// * operators/operator_sdk/osdk-managing-bundle-images.adoc
+
+[id="olm-bundle-format_{context}"]
+= Bundle Format
+
+The _Bundle Format_ for Operators is a new packaging format introduced by the
+Operator Framework that is supported starting with {product-title} 4.5. To
```

Comment:
Support statement might be confusing - bundles install fine back to 4.3

Reply:
In downstream OCP at least, we danced around it as "Dev Preview" in 4.4 for the first time and are only saying GA as of 4.5... Maybe I'll just avoid mentioning a version here.